modules/mri_upload/test/mri_uploadTest.php

```diff
@@ -8,10 +8,5 @@ function testMriUploadDoespageLoad()
         $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
         $this->assertContains("mri_upload", $bodyText);
     }
-
-    public function tearDown()
-    {
-        $this->webDriver->quit();
-    }
```

Comment:
Can you rebase this branch? This (and some of the changes below) are already merged and makes it harder to see the diff